package.json

```diff
@@ -69,10 +68,13 @@
     "@types/yamljs": "^0.2.30",
     "@types/yargs": "^8.0.3",
     "body-parser": "^1.18.3",
+    "moleculer": "^0.13.7",
```

Comment:
I think that if you could submit this PR without bringing in `moleculer`, `moleculer-web`, and `moleculer-decorates` then I'd be more likely to approve it. But I don't understand why we're bringing in so many additional dependencies.